### ISSUE TITLE
Documentation improvement

### ISSUE THREAD

**Issue by peuter** [CometVisu/CometVisu#416]
Basics:
- [ ] Transfer missing content from wiki (+ add `@since` information to widget pages)
- [ ] Complete documentation colaboration part, describe scaffolding
- [ ] Extend scaffolding to plugins
- [ ] Fill the gaps, work through the TODOs: http://test.cometvisu.org/CometVisu/de/manual/colab/todos.html
- [ ] refactor structure to `/<language>/<version>/(manual|api|...)`

New content:
- [ ] Start english documentation (mix content from wiki + from widget api doc)
- [ ] add some openHAB1/2 specific documentation (installation, configuration, diagrams)
- [ ] 

Nice to have:
- [ ] shared design template for all documentation parts (manual, api) + homepage
- [ ] add version links (like in read-the-docs)
- [ ] improve documentation helper:
  - [ ] use virtualenc + setuptools
  - [ ] more assistance
  - [ ] add simple git repo auto-setup
  - [ ] add git workflow (start doc: update local repo, create new branch, save doc: commit + push, end doc: create pull request)

**Comment by ChristianMayer** [comment]
Any (which?) points that should be done before the 0.10.0 release? 
(I.e. which points would block a 0.10.0 release?)

**Comment by peuter** [comment]
IMHO There is no showstopper here, of course the documentation is not complete, buts its more than the old one, so not blocking a release.

**Comment by JonasArnold** [comment]
In Version 0.10.2 the documentation of the startpage parameterr is wrong. With 0.10.1 the startpage URL parameter was changed so that the name of the page can be used instead of the ID. But the 0.10.2 manual still describes the old way (to use the ID and how to get it).

**Comment by peuter** [comment]
Well the old way should still work. So its not wrong, just incomplete.

**Comment by JonasArnold** [comment]
Actually the old way didn't work for me because my browser does not display the target URL of the Navbar-element (the `javascript:templateEngine.scrollToPage('id_0_33')` thing). Maybe this is a problem with my browser..

**Comment by peuter** [comment]
That way might indeed be wrong described. You can just open the Developer Tools of your Browser and look into the DOM-Tree. There you can find all ids.
